src/shared/codelens/localLambdaRunner.ts

```diff
@@ -251,14 +251,14 @@ export class LocalLambdaRunner {
             JSON.stringify(this.getEnvironmentVariables(config))
         )
 
-        const command = new SamCliLocalInvokeInvocation(
-            LocalLambdaRunner.TEMPLATE_RESOURCE_NAME,
-            samTemplatePath,
+        const command = new SamCliLocalInvokeInvocation({
+            templateResourceName: LocalLambdaRunner.TEMPLATE_RESOURCE_NAME,
+            templatePath: samTemplatePath,
             eventPath,
```

Comment:
`eventPath` and `environmentVariablePath` should be assigned to fields too

Reply:
The fields are the same name, so it isn't required.